docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-importing-tokens.mdx

```diff
@@ -0,0 +1,67 @@
+---
+keywords: [beginner, governance, tutorial, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import '/src/components/CenterImages/center.scss';
+
+# Import ICRC tokens
+
+<MarkdownChipRow labels={["Beginner", "Governance", "Tutorial"]} />
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zcXmarBk2L4?si=OIxq9LwY0DLzHvE1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## Overview
+
+The NNS dapp focuses on providing a simple user interface where community members can interact with different on-chain governance systems deployed on the Internet Computer.
+
+To extend wallet capabilities, users of the NNS dapp can now import any token that supports ICRC-1 no longer limited to ICP and SNS governance tokens. This includes all ICRC-2 and ICRC-3 tokens.
+
+## How to find canister IDs
+
+There are several ways to find ledger canister IDs of ICRC-1 tokens. The tutorial video above mentions two. This is a more comprehensive list.
+
+- [ICPSwap](https://info.icpswap.com/swap): Curated list of tokens traded on ICPSwap updated by the ICPSwap SNS DAO.
+- [ICP Tokens](https://icptokens.net/): Curated list of tokens traded on ICP-based DEXs updated by ICP Tokens.
+- [ICP Dashboard](https://dashboard.internetcomputer.org/): List of all chain-key tokens (under Chain Fusion menu option) updated automatically.
+- [nftGeek](https://t5t44-naaaa-aaaah-qcutq-cai.raw.icp0.io/tokens): Curated list of tokens traded on ICP-based DEXs updated by nftGeek.
+- [ICPEx](https://icpex.org/dashboard): Curated list of tokens traded on ICPEx updated by ICPEx.
+- [CoinMarketCap](https://coinmarketcap.com/): The ICRC tokens that are listed on CoinMarketCap provide their ledger canister ID under Contract.
+- [CoinGecko](https://coingecko.com/): The ICRC tokens that are listed on CoinGecko provide their ledger canister ID under Contract.
+
+
+## How to import ICRC tokens
+
+First, find the ledger canister ID of the token you want to import to your NNS dapp wallet. The list above covers most tokens deployed on the Internet Computer. Once you have its ledger canister ID, you may optionally look for its index canister to display the transaction history.
```

Comment:
would it make sense to add a sentence on top what it means to import a token?
What can you do with it afterwards? are there risks?
Maybe this can actually be part of the overview?